presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improved performance of sum and average aggregations on decimals.
+* Support for LocationProvider supplied paths in Iceberg connector.
+* Column comments now available in JDBC connector.
+* Hidden columns ``file_modified_time`` and ``file_size`` added to Hive connector.
+* Partition lease support added to Hive connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix the Aggregation IF to FILTER rule by not unwrapping the IF expression inside the Aggregation. The rule now rewrites ``AGG(IF(condition, expr))`` to ``AGG(IF(condition, expr)) FILTER (WHERE condition)``.
+* Fix marking coordinator active so it is only done when the required minimum resource managers and required minimum coordinators are available in a cluster. Required resource managers can be configured by ``cluster.required-resource-managers-active`` (default value: 1, disaggregated coordinator only). Required coordinators can be configured by ``cluster.required-coordinators-active`` (default value: 1).
+* Improve performance of sum and average aggregations on decimals.
+* Add additional runtime stats for ``ScanFilterAndProject`` operator. Previously only ``TableScanOperator`` was exposed.
+* Disable ``optimizer.aggregation-if-to-filter-rewrite-enabled`` by default.
+
+SPI Changes
+___________
+* Add ``Identity`` field to ``checkCanSetUser`` for finer granularity of system access control.
+
+Hive Changes
+____________
+* Add hidden column ``$file_modified_time``.
+* Add hidden column ``$file_size``.
+* Add support for partition leases. Configuration property ``hive.partition-lease-duration`` and session property ``partition_lease_duration`` can be used to specify the duration of a lease (e.g. "4h") or disable partition leases ("0h").
+* Add CPU usage of file metadata reading to runtime statistics.
+
+Iceberg Changes
+_______________
+* Add LocationProvider support to Iceberg connector so files are now written to paths defined by Iceberg's LocationProvider instead of a hard-coded table root directory.
```

Comment:
Is LocationProvider a code or some other concept?

Reply:
A location providers is part of your iceberg configuration. Since it is pluggable we don't really know which one a user picked we just know that we will now respect the paths the location provider specifies.

How do you think it should be phrased?